exercises/protein-translation/canonical-data.json

```diff
@@ -0,0 +1,143 @@
+          "property": "translate",
+          "strand": "UUA",
+          "expected": ["Leucine"]
+        },
+        {
+          "description": "Leucine RNA sequence 2",
+          "property": "translate",
+          "strand": "UUG",
+          "expected": ["Leucine"]
+        },
+        {
+          "description": "Serine RNA sequence 1",
+          "property": "translate",
+          "strand": "UCU",
+          "expected": ["Serine"]
+        },
+        {
+          "description": "Serine RNA sequence 2",
+          "property": "translate",
+          "strand": "UCC",
+          "expected": ["Serine"]
+        },
+        {
+          "description": "Serine RNA sequence 3",
+          "property": "translate",
+          "strand": "UCA",
+          "expected": ["Serine"]
+        },
+        {
+          "description": "Serine RNA sequence 4",
+          "property": "translate",
+          "strand": "UCG",
+          "expected": ["Serine"]
+        },
+        {
+          "description": "Tyrosine RNA sequence 1",
+          "property": "translate",
+          "strand": "UAU",
+          "expected": ["Tyrosine"]
+        },
+        {
+          "description": "Tyrosine RNA sequence 2",
+          "property": "translate",
+          "strand": "UAC",
+          "expected": ["Tyrosine"]
+        },
+        {
+          "description": "Cysteine RNA sequence 1",
+          "property": "translate",
+          "strand": "UGU",
+          "expected": ["Cysteine"]
+        },
+        {
+          "description": "Cysteine RNA sequence 2",
+          "property": "translate",
+          "strand": "UGC",
+          "expected": ["Cysteine"]
+        },
+        {
+          "description": "Tryptophan RNA sequence",
+          "property": "translate",
+          "strand": "UGG",
+          "expected": ["Tryptophan"]
+        },
+        {
+          "description": "STOP codon RNA sequence 1",
+          "property": "translate",
+          "strand": "UAA",
+          "expected": []
+        },
+        {
+          "description": "STOP codon RNA sequence 2",
+          "property": "translate",
+          "strand": "UAG",
+          "expected": []
+        },
+        {
+          "description": "STOP codon RNA sequence 3",
+          "property": "translate",
+          "strand": "UGA",
+          "expected": []
+        },
+        {
+          "description": "Translate RNA strand into correct protein list",
+          "property": "translate",
+          "strand": "AUGUUUUGG",
+          "expected": ["Methionine","Phenylalanine","Tryptophan"]
+        },
+        {
+          "description": "Translation stops if STOP codon at end of sequence",
+          "property": "translate",
+          "strand": "AUGUUUUAA",
+          "expected": ["Methionine","Phenylalanine"]
+        },
+        {
+          "description": "Translation stops if codon in middle of sequence",
+          "property": "translate",
+          "strand": "UGGUGUUAUUAAUGGUUU",
```

Comment:
Put some thought put into which sequences are easiest for humans to read and distinguish.